lib/shopify-cli/shopifolk_check.rb

```diff
@@ -0,0 +1,45 @@
+require 'fileutils'
+
+module ShopifyCli
+  class ShopifolkCheck
```

Comment:
Should this class have a simpler interface? We really only need to access a single method on it. For instance the class could just be called Shopifolk and then we call `Shopifolk.check`

Also, you will need to document this class as the other public classes because it may get used by other teams as well.

Reply:
Yeah, let's make all the other methods private.